.changeset/nine-windows-join.md

```diff
@@ -0,0 +1,17 @@
+---
+"@makeswift/runtime": patch
+---
+
+BREAKING: When registering component icons, use the `ComponentIcon` enum (available under `@makeswift/runtime/state/components-meta`) instead of the original string values. Below is a table of the deprecated string values and their new enum equivalent:
```

Comment:
We should not export internals like the `component-meta` this way. Instead the export should be available from `@makeswift/runtime` directly:

```ts
import { ComponentIcon } from '@makeswift/runtime'
```